core/src/main/scala/cats/data/NonEmptyList.scala

```diff
@@ -100,6 +100,15 @@ final case class NonEmptyList[+A](head: A, tail: List[A]) {
   def prepend[AA >: A](a: AA): NonEmptyList[AA] =
     NonEmptyList(a, head :: tail)
 
+  /**
+    * Alias for append
+    */
+  def :+[AA >: A](a: AA): NonEmptyList[AA] =
```

Comment:
this method should be `final`

Reply:
sure but I would prefer we do that in a separate PR as there is no final method in NEL yet